### ISSUE TITLE
There is no tag singal `property::gap`

### ISSUE THREAD

**Issue by actionless** [awesomeWM/awesome#3692]

**Comment by kosorin** [comment]
Isn't `property::useless_gap` synonym?

**Comment by Elv13** [comment]
@actionless Is it documented somewhere it exists? The doc mentions `property::useless_gap` (click "Click to display more" in the doc to see the signal information)

**Comment by actionless** [comment]
mb then it's a bug with rendering the doc TOC:

![2022-10-04--1664890623_1073x431_scrot](https://user-images.githubusercontent.com/1655669/193833985-7dab6bd0-f55d-438f-bb60-0efa7c37cf65.png)

**Comment by actionless** [comment]
and also yeah that's confusing that property itself is called 'gap' and signal is called `property::useless_gap` not `property::gap` as for other properties' signals

**Comment by Elv13** [comment]
Yes, I should probably cleanup the last few `property::` signals which use `@signal` instead of `@emits`/`@propsemit`. I collapsed the property and property signal documentation into one because "signal for the ***** property` wasn't very useful and very repetitive. At least with the collapsed doc, there's a single place to talk about the property. I am inclined to do it for theme variables too, eventually. Right now theme variable docs is mostly "theme variable for property *****" and that's not very useful compared to a consolidated and complete doc block.
